src/Angor/Client/NetworkConfiguration.cs

```diff
@@ -44,6 +46,16 @@ public Network GetNetwork()
         return currentNetwork;
     }
 
+    public bool GetDebugMode()
```

Comment:
no you didn't understand me correctly, you cannot inject the `ISyncLocalStorageService` here because this class is not loaded in the Dependency Injection service.

What I ment it create a param called `DebugMode` and the method `GetDebugMode` and `SetDebugMode` will set that param